conf/ocsci/multicluster_mode_mdr.yaml

```diff
@@ -0,0 +1,2 @@
+ENV_DATA:
+  mdr_scenario: True
```

Comment:
I think it's better if we can specify MDR scenario using `multicluster_mode`. For reference see [multicluster_mode_rdr.yaml](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/multicluster_mode_rdr.yaml)